.github/workflows/gh_actions_pr.yaml

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2012-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+name: Docker Build
+
+on: pull_request
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout che-machine-exec source code
+      uses: actions/checkout@v2
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+    - name: Build and push nightly
+      uses: docker/build-push-action@v2
+      with:
+        file: build/dockerfiles/Dockerfile
+        platforms: linux/amd64,linux/ppc64le,linux/s390x
+        push: false
+        tags: quay.io/eclipse/che-machine-exec:nightly
```

Comment:
When it's used? Can we just omit it? Or add tag like `pr-check` because it's not nightly at all.

Reply:
Ok, I'll change that.

Reply:
But than we should change "nightly"  tag for che-plugin-registry too

Reply:
And che-server: https://github.com/eclipse/che/blob/master/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L678
https://github.com/eclipse/che/blob/master/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L658

Reply:
In this case no. This job is just a check in PRs if machine-exec image can built it.